ui/perfherder/tests/TestsView.jsx

```diff
@@ -71,7 +71,7 @@ class HeathView extends React.PureComponent {
     }
   };
 
-  getTestSuiteHealthData = async params => {
+  getTestSuiteData = async params => {
```

Comment:
Probably could use the `fetch` prefix here, instead of `get`, so it's clearer what this method does.